app/src/main/java/io/apicurio/registry/ccompat/dto/CompatibilityLevelParamDto.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.ccompat.dto;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+
+/**
+ *
+ * @author Carles Arnal <carnalca@redhat.com>
+ */
+@JsonAutoDetect(isGetterVisibility = NONE)
+@NoArgsConstructor // required for Jackson
+@AllArgsConstructor
+@Getter
+@EqualsAndHashCode
+@ToString
+public class CompatibilityLevelParamDto{
+
+    @JsonProperty("compatibilityLevel")
+    private String compatibility;
```

Comment:
Should this instead share the `Level` enum?

Reply:
I don't think so, this is just being used for return purposes at API level.